src/web/mjs/connectors/KomikTap.mjs

```diff
@@ -7,11 +7,11 @@ export default class KomikTap extends WordPressMangastream {
         super.id = 'komiktap';
         super.label = 'KomikTap';
         this.tags = [ 'manga', 'webtoon', 'hentai', 'indonesian' ];
-        this.url = 'https://komiktap.in';
+        this.url = 'https://92.87.6.124/';
```

Comment:
Nope. Sheep told me not to put ip as host, so i solve it as runtime, since komiktap.in redirector to the ip.
You can remove this file from your PR as its already ready

Reply:
i have tried it and it works fine

Reply:
Its not a matter of working, its a matter of what is acceptable in the code.

Reply:
Anyway its solved by an incoming PR, pls remove those changes.

Reply:
Okay , sorry 👍